src/middleware/node/types.ts

```diff
@@ -9,8 +9,4 @@ import { Logger } from "../../createLogger";
 export type MiddlewareOptions = {
   path?: string;
   log?: Logger;
-  onUnhandledRequest?: (
```

Comment:
Can you remove the type declarations, and the associated comments that are now unused

Reply:
Thanks. Done.